repos/spack_repo/builtin/packages/gcc/package.py

```diff
@@ -425,6 +425,23 @@ class Gcc(AutotoolsPackage, GNUMirrorPackage, CompilerPackage):
     # https://github.com/iains/gcc-12-branch/issues/6
     conflicts("@:12", when="%apple-clang@14:14.0")
 
+    # Applies
+    # https://github.com/gcc-mirror/gcc/commit/ea2798892de373b14f9fc7ae8a0d820eaddca98c,
+    # which fixes an incorrectly applied fixincludes rule for pthread.h, making
+    # the installed GCC not portable across different glibc versions. Original
+    # GCC bug report: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=118009. For
+    # GCC 15 we can directly use the upstream patch. For GCC 12-14 the patch
+    # has been backported. For GCC 11 the fix should actually be applied and
+    # older versions have not been tested.
```

Comment:
@msimberg not clear to me what this means, in particular about GCC11.

I mean that 

> I'm not applying the patch to 11 since apparently the "fixed" include should actually be used for that version (see GCC commit description)

is a bit more clearer.

Reply:
My confusion comes from the fact that we are talking about a fix of a fix 😅

Reply:
Yeah, thanks for the comment @albestro! I admit that's not very clear :stuck_out_tongue: 

What do you think about this slightly reworded comment from the PR description:
```suggestion
    # has been backported. The patch is not applied to GCC 11 since the "fixinclude"
    # is in fact needed for that version (see GCC commit description). Older versions
    # have not been checked or tested.
```
?

Reply:
🔝